pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -660,11 +660,6 @@ def _check_and_init_precision(self) -> PrecisionPlugin:
     def _validate_precision_choice(self) -> None:
         """Validate the combination of choices for precision, AMP type, and accelerator."""
         # TODO: change exception type to ImpactableConfigurationException
```

Comment:
Remove this TODO?

```suggestion
```

Reply:
This is still a valid TODO i think, @justusschock has a issue about it

Reply:
Oh! The one you mentioned above :)

Reply:
Yes, but you moved the exception but not the TODO

Still, I think `ValueError` is fine.

Reply:
Sounds good :)